dataikuapi/dss/savedmodel.py

```diff
@@ -69,6 +69,22 @@ def set_active_version(self, version_id):
         self.client._perform_empty(
             "POST", "/projects/%s/savedmodels/%s/versions/%s/actions/setActive" % (self.project_key, self.sm_id, version_id))
 
+    def delete_versions(self, versions, remove_intermediate=True):
+        """
+        Delete version(s) of the saved model
+
+        :param versions: list of versions to delete
+        :type versions: list[str]
+        :param remove_intermediate: also remove intermediate versions (default: True). In the case of a partitioned
+        model, an intermediate version is created every time a partition has finished training.
+        :type remove_intermediate: bool
+        """
+        self.client._perform_empty(
+            "POST", "/projects/%s/savedmodels/%s/actions/delete-versions" % (self.project_key, self.sm_id),
+            params={
+                'versions': versions,
```

Comment:
In those helpers, we tend to be liberal in what we accept, so we can do a special treatment if `versions` is a string\* and automatically wrap it in an array.
\* Simpler to test as "not an array" because string is complicated (`unicode` or `str` in py2, `str` or `bytes` in py3…).

Reply:
I just checked: the JSON parser is smart enough to convert a string param to a list of strings with a single instance.
So, no need to complexify the client, IMHO.

Reply:
Oh, nice